SPECS/k3s/k3s.spec

```diff
@@ -1,21 +1,21 @@
 Summary:        Lightweight Kubernetes
 Name:           k3s
-Version:        1.25.0
+Version:        1.25.5
```

Comment:
I would request iterating on a single PR instead of closing one and opening a new one. (It becomes a little difficult to review)

Review Context:
- https://github.com/microsoft/CBL-Mariner/pull/5210
- https://github.com/microsoft/CBL-Mariner/pull/5209

Reply:
Sure, consider this as the final PR for this version upgrade.